### PR TITLE
ADD container kill in README

### PR DESCRIPTION
**Pull request by fewdan**
### What problem does this PR solve?
ADD container kill in README

### Check List

Tests 

 - No code

Code changes

 - NO

Side effects

 - NO

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,7 +194,7 @@ metadata:
   name: pod-failure-example
   namespace: chaos-testing
 spec:
-  action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure
+  action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure/container-kill
```
Comment (zhouqiang-cl) [inline README.md]: Did we need to specify container name?

Comment (fewdan) [reply README.md]: Yes. But the example here the action is pod-failure, so there is no information describing the container name.

Comment (zhouqiang-cl) [reply README.md]: so. maybe we can remote container-kill here

File: README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Chaos Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/doc
 
 - pod-kill: The selected pod is killed (ReplicaSet or something similar may be needed to ensure the pod will be restarted).
 - pod-failure: The selected pod will be unavailable in a specified period of time.
+- container-kill: The selected container is killed in the selected pod.
```
Comment (zhouqiang-cl) [inline README.md]: how about also add time skew?

**Comment by YangKeao** [comment]
Great! Could we also modify ROADMAP?

**Comment by zhouqiang-cl** [comment]
> Great! Could we also modify ROADMAP?

Agree. we should modify ROADMAP

**Comment by fewdan** [comment]
> Great! Could we also modify ROADMAP?
@YangKeao
Are you referring to " Support BPF injection for time skew chaos. "  and "Add container kill/failure chaos" in ROADMAP ?

File: README.md [hunk 3]
```diff
@@ -36,9 +36,11 @@ Chaos Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/doc
...
 - netem chaos: Network chaos such as delay, duplication, etc.
 - network-partition: Simulate network partition.
 - IO chaos: Simulate file system faults such as I/O delay, read/write errors, etc.
+- time chaos: The selected pod will be injected time offset.
```
Comment (cwen0) [inline README.md]: s/time offset/timing skew/g?

Comment (fewdan) [reply README.md]: done

**Pull request merged**